packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -322,10 +326,37 @@ async function findAndInvalidateLinks(
       return;
     }
 
+    const patient = await getPatientOrFail({ cxId, id: patientId });
+    const cwAccess = await getCWAccessForPatient(patient);
+    if (cwAccess.error != null) throw new MetriportError(cwAccess.error);
```

Comment:
Prob need more than the value inside `.error` since that's just a "key" kinda thing, eh?

Reply:
It contains a message in the error such as "missing-facility-id"

Reply:
I know, that's what I meant with "key". When getting an alert, this won't add context, and prob more useful on Sentry's extra/additionalInfo than on the event title, IMO.

I'd go with something like

```ts
...MetriportError("Error invalidating CW link", undefined, { reason: cwAccess.error);
```

Thoughts?